Extensions/Effects/JsExtension.js

```diff
@@ -897,10 +915,11 @@ module.exports = {
         .setDescription('Blur intensity of the vignette')
     );
     oldFilmProperties.set(
-      'animated',
-      new gd.PropertyDescriptor(/* defaultValue= */ 'true')
-        .setLabel(_('Animated (Enable animations)'))
-        .setType('boolean')
+      'animationFrequency',
+      new gd.PropertyDescriptor(/* defaultValue= */ '0')
```

Comment:
Same, let's make a default value that was equivalent to animated = true, i.e: 60